aws-ts-assume-role/assume-role/Pulumi.yaml

```diff
@@ -0,0 +1,12 @@
+name: assume-role
+runtime: nodejs
+description: Demonstrate use of AWS AssumeRole Functionality
+template:
```

Comment:
I _think_ this can be removed (an instance of https://github.com/pulumi/pulumi/issues/2044).

Reply:
Ah, I was misunderstanding the purpose of a template - I've removed them now.